examples/hello_triangle_c/main.c

```diff
@@ -1,6 +1,10 @@
 #include <stdio.h>
 #include "./../../wgpu-bindings/wgpu.h"
 
+#define STAGES_LENGTH       (2)
```

Comment:
:+1: I guess we'll probably still use `malloc`/`memset` for complex examples, but I think these `#define`s are fine here